base/shell/explorer/startup.cpp

```diff
@@ -505,21 +510,23 @@ INT ProcessStartupItems(VOID)
     return res ? 0 : 101;
 }
 
-BOOL DoFinishStartupItems(VOID)
+INT ProcessStartupItems(VOID)
```

Comment:
I wonder whether this could be made as an inline function (in the header), or the `ProcessStartupItems(FALSE)` directly called where it is called (line 169 in explorer.cpp)